axis.go

```diff
@@ -248,9 +248,9 @@ func (a *horizontalAxis) GlyphBoxes(*Plot) (boxes []GlyphBox) {
 		w := a.Tick.Label.Width(t.Label)
 		box := GlyphBox{
 			X: a.Norm(t.Value),
-			Rectangle: draw.Rectangle{
-				Min: draw.Point{X: -w / 2},
-				Max: draw.Point{X: w / 2},
+			Rectangle: vg.Rectangle{
```

Comment:
That's a pretty standard git workflow.

Next time, I would just not send the follow up PR until the first is in, because github has no way to tell the reviewers what is going on.